lib/internal/bootstrap/primordials.js

```diff
@@ -23,6 +36,18 @@ function copyProps(src, dest) {
   }
 }
 
+function copyPrototype(src, dest) {
+  for (const key of Reflect.ownKeys(src)) {
+    if (!Reflect.getOwnPropertyDescriptor(dest, key)) {
+      const desc = Reflect.getOwnPropertyDescriptor(src, key);
+      if (typeof desc.value === 'function') {
+        desc.value = uncurryThis(desc.value);
```

Comment:
I was talking about this, as this affect all prototype methods, not just the Function ones.

Reply:
It is already what was done manually in all the files I changed (either with call.bind or uncurryThis). This is centralizing the uncurrying to this file. If we want to use the methods safely, we have to call them without going through the prototype.